### DISCUSSION TITLE
Parallax: scrollTo(offset) with scroll down movement

### DISCUSSION THREAD

**Discussion by danielbeckera** [pmndrs/react-spring#1975] [Support]
I'm trying to trigger the scrollTo() method with scroll movement (mouse wheel down) on Parallax but I can't find any solution to this.
It only works with onClick, but no with onScroll, can you guys give me an example on how to do this using scroll?
I would like to jump between offsets when scroll is triggered.

I saw this topic https://github.com/pmndrs/react-spring/issues/1953 here, but I didnt understand this part:

> We've seen a handful of Discussions/Questions regarding adding scroll listeners to Parallax recently. As it stand right now, the method to do that is pretty boilerplate-y, to say the least. A user has to:

> -  attach a ref to Parallax
> - get the ref of the container from that ref (ref.current.container.current)
> - add an event listener to that ref

Thanks in advance.

**Comment by joshuaellis** [comment]
Sorry, i'm just trying to understand your question, you want to trigger a scrollTo event when someone is scrolling, isn't that counter intuitive? And additionally, go against an expected behaviour? because if you scroll and `scrollY` was `10` you'd instead rather it jumped to `100` but what would happen with the inertia etc?

**Reply by eeshankeni** [reply]
I would also like to achieve this functionality

Essentially letting framer motion animations and stuff completely animate with an automatic scroll to specific pages or even offsets maybe.

These animations are interpolated with the scrolling of the entire root parallax component anyway. Thank you!

**Comment by danielbeckera** [comment]
Hi,
What I want is a way to jump between offsets as soon as the user scrolls down/up, just like using:
`onClick={( ) => { ref.current.scrollTo(1) }}`
But with scroll, so the user goes immediately to the offset instead of having to scroll multiple times to get there (right now, for the user to see the page 2 text, he needs to scroll several times, I would like to trigger the scrollTo as soon as user scrolls). 
I hope I was clear this time, if you need more information I'll gladly provide it.
Thanks!
